index.bs

```diff
@@ -787,6 +792,49 @@ following steps:
 
 </div>
 
+<h3 id="bounce-tracking-mitigations-automation">User Agent Automation</h3>
+
+For the purposes of user-agent automation and application testing, this
+document defines a number of [extension commands] for the [[WebDriver]]
+specification.
+
+<h4 id="run-bounce-tracking-mitigations-command">Run Bounce Tracking Mitigations</h4>
+
+<figure id="table-webdriver-run-bounce-tracking-mitigations-command" class="table">
+    <table class="data">
+        <thead>
+            <tr>
+                <th>HTTP Method</th>
+                <th>URI Template</th>
+            </tr>
+        </thead>
+        <tbody>
+            <tr>
+                <td>DELETE</td>
+                <td>`/session/{session id}/storage/run_bounce_tracking_mitigations`</td>
+            </tr>
+        </tbody>
+    </table>
+</figure>
+
+The [=remote end steps=] are:
+
+1. If no Bounce Tracking Mitigation Service is currently running, return a [=error|WebDriver error=] with
+    [=error code=] [=unsupported operation=].
+
+1. If the [=stateful bounce tracking map=] contains a malformed host, return a [=error|WebDriver error=] with
+    [=error code=] [=unknown error=].
+
+1. Let |list| be a new list of the [=site=] [=hosts=] in the [=stateful bounce tracking map=].
+
+1. Set [=bounce tracking grace period=] to 0 seconds.
+
+1. Run the [=bounce tracking timer=] algorithm.
+
+1. Set [=bounce tracking grace period=] back to its original value.
```

Comment:
It would be slightly better to save the original value in a variable and then set it back explcitly.

Reply:
I've moved to using a temp variable in the latest commit. How do you feel about the approach?